parts/inc/magic

```diff
@@ -552,3 +622,68 @@ ok(!Devel::PPPort::SvVSTRING_mg(4711));
 my $foo = 'bar';
 ok(Devel::PPPort::sv_magic_portable($foo));
 ok($foo eq 'bar');
+
+if ( "$]" lt '5.007003' ) {
+    skip 'skip: no SV_NOSTEAL support', 0 for 1..22;
+} else {
+    tie my $scalar, 'TieScalarCounter', 10;
+    my $fetch = $scalar;
+
+    ok tied($scalar)->{fetch}, 1;
```

Comment:
should not these tests use `is` instead of `ok`?

Reply:
Devel::PPPort does not use `Test::Simple`, but rather `Test` module which has different API. See:
https://github.com/Dual-Life/Devel-PPPort/pull/128/files#diff-e998a77b1ca4b0163f81709c60da878fR29
And it has only 3 argument `ok` function, see:
https://github.com/Dual-Life/Devel-PPPort/blob/master/t/testutil.pl#L20

Reply:
This is in order to maintain compatibility with perls < 5.6.  However, we could write our own `is`, `isnt`, `like` functions in a local library (wrapping Test::ok) that provide the same APIs as Test::More's functions, to make it easier to write (and read) the tests.

Reply:
This is not the first time that I haven't received a notification from github.  How can I track the cause of this down.  Is it github, is it cpan.org?.

Having stumbled upon this, I agree with ether's sentiment.  Who is going to write this wrapper?

Reply:
notifications are coming from GitHub itself, you should check your subscription  status to the repo but also to the issue itself

Reply:
I've opened #130 for tracking this wish list, I would say the first that want to work on it should assign the task to himself

Reply:
Ok, using new test wrapper would mean to rewrite all tests... I'm not planning to do it as it is a big work right now and we can leave without it.